src/shared-components/dropdown/index.js

```diff
@@ -40,9 +44,8 @@ class Dropdown extends React.Component {
   };
 
   onOptionClick = event => {
-    if (event.target.hasAttribute('disabled')) {
-      return;
-    }
+    if (event.target.hasAttribute('disabled')) return;
```

Comment:
prefer the block over not. Whitespace and new line is removed when minified so we can optimize for human readability

Reply:
👍